micro_ros_diagnostic_bridge/src/micro_ros_diagnostic_bridge/micro_ros_diagnostic_bridge.cpp

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2021 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository https://github.com/microros/system_modes
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "micro_ros_diagnostic_bridge/micro_ros_diagnostic_bridge.hpp"
+
+#include <string>
+#include <memory>
+#include <utility>
+
+#include <rcl/error_handling.h>
+
+#include <rclcpp/parameter.hpp>
+#include <rclcpp/parameter_map.hpp>
+#include <rcl_yaml_param_parser/parser.h>
+
+using micro_ros_diagnostic_msgs::msg::MicroROSDiagnosticStatus;
+using diagnostic_msgs::msg::DiagnosticStatus;
+
+MicroROSDiagnosticBridge::MicroROSDiagnosticBridge(const std::string & path)
+: Node("micro_ros_diagnostic_bridge"),
+  logger_(rclcpp::get_logger("MicroROSDiagnosticBridge"))
```

Comment:
Not sure whether `logger_ = get_logger()` in the body of the ctor would be better ...

Or remove `logger_` at all as it does not seem to be used.